test/integration/viz/policy/policy_test.go

```diff
@@ -26,10 +26,22 @@ func TestMain(m *testing.M) {
 //////////////////////
 
 func TestPolicy(t *testing.T) {
+	for _, policy := range []string{
+		"emoji-policy",
+		"emoji-policy-authorization-policy",
+		"emoji-policy-authorize-all",
+	} {
+		t.Run(policy, func(t *testing.T) {
+			runWithPolicyManifests(t, policy)
+		})
+	}
+}
```

Comment:
Hm, is it necessary to add viz tests? Are we actually testing the viz behavior?

The `policy-test` directory already has admission tests and e2e tests that I think we can extend to test namespace behavior